cmpserver/plugin/plugin.go

```diff
@@ -286,18 +286,19 @@ func (s *Service) matchRepositoryGeneric(stream MatchRepositoryStream) error {
 	return nil
 }
 
-func (s *Service) matchRepository(ctx context.Context, workdir string, envEntries []*apiclient.EnvEntry) (bool, error) {
+func (s *Service) matchRepository(ctx context.Context, workdir string, envEntries []*apiclient.EnvEntry) (bool, bool, error) {
```

Comment:
Can you add names to the return types for readability?

Reply:
updated